### ISSUE TITLE
NFT transfers bugs

### ISSUE THREAD

**Issue by andreaarmanni** [oceanprotocol/market#1707]
I transfered a [data NFT](https://market.oceanprotocol.com/asset/did:op:bacc5d13e2eea5f4729ad12c05a543646476e8517bc27721575622f4cec23f7c) to an ENS wallet (datachallenges.eth) using OpenSea. The [tx](https://etherscan.io/tx/0x6bbc3355bc31fcba61b458d6b5d39c9031a6fe125dac6e57101d4780109e0c69) took approx 15 min and eventually was confirmed. However, I found the following issues:

- [ ] Ocean Market doesn’t seem to update the ownership of the data NFT. In fact, the data NFT is still linked to the previous wallet, both in /profile/ and in /asset/
- [ ] The frontend shows that the previous owner still has editing rights 
- [ ] The data NFT can't be indexed by OpenSea after it is transferred
- [ ] Ocean Market doesn’t show the data NFT under the new owner’s profile

[Slack conversation](https://oceanprotocol.slack.com/archives/CNDG1849Z/p1664009637346639)

**Comment by kremalicious** [comment]
Frontend only displays what Aquarius gives us and until recently Aquarius did not capture any transfer events causing this issue so all on Aquarius side. Fixed in https://github.com/oceanprotocol/aquarius/pull/874 which has not been deployed yet.

**Comment by andreaarmanni** [comment]
Thanks for the update. What's the expected timeline for this?

**Comment by kremalicious** [comment]
Aquarius is at latest versions now so https://github.com/oceanprotocol/aquarius/pull/874 should be fixed. Reopen this issue if there's still problems with transfer

**Comment by andreaarmanni** [comment]
I've reopened this issue as the same bugs have remained even after the latest aquarius update. More details 👇 

I transfered a [data NFT](https://market.oceanprotocol.com/asset/did:op:bacc5d13e2eea5f4729ad12c05a543646476e8517bc27721575622f4cec23f7c) to an ENS wallet (datachallenges.eth) using OpenSea. The [tx](https://etherscan.io/tx/0x6bbc3355bc31fcba61b458d6b5d39c9031a6fe125dac6e57101d4780109e0c69) took approx 15 min and eventually was confirmed. However, I found the following issues:

- [ ] Ocean Market doesn’t seem to update the ownership of the data NFT. In fact, the data NFT is still linked to the previous wallet, both in /profile/ and in /asset/
- [ ] The frontend shows that the previous owner still has editing rights 
- [ ] The data NFT can't be indexed by OpenSea after it is transferred
- [ ] Ocean Market doesn’t show the data NFT under the new owner’s profile

[Slack conversation](https://oceanprotocol.slack.com/archives/CNDG1849Z/p1664009637346639)
